Assets/MixedRealityToolkit/_Core/Inspectors/Profiles/MixedRealitySpatialAwarenessProfileInspector.cs

```diff
@@ -0,0 +1,144 @@
+        private readonly GUIContent physicsLayerContent = new GUIContent("Physics Layer");
+        private readonly GUIContent trianglesPerCubicMeterContent = new GUIContent("Triangles/Cubic Meter");
+        private readonly GUIContent useSystemContent = new GUIContent("Use System");
+
+        private void OnEnable()
+        {
+            if (!CheckMixedRealityManager(false))
+            {
+                return;
+            }
+
+            // General settings
+            startupBehavior = serializedObject.FindProperty("startupBehavior");
+            observationExtents = serializedObject.FindProperty("observationExtents");
+            updateInterval = serializedObject.FindProperty("updateInterval");
+
+            // Mesh settings
+            useMeshSystem = serializedObject.FindProperty("useMeshSystem");
+            meshPhysicsLayer = serializedObject.FindProperty("meshPhysicsLayer");
+            meshLevelOfDetail = serializedObject.FindProperty("meshLevelOfDetail");
+            meshTrianglesPerCubicMeter = serializedObject.FindProperty("meshTrianglesPerCubicMeter");
+            meshRecalculateNormals = serializedObject.FindProperty("meshRecalculateNormals");
+            meshDisplayOption = serializedObject.FindProperty("meshDisplayOption");
+            meshMaterial = serializedObject.FindProperty("meshMaterial");
+            meshOcclusionMaterial = serializedObject.FindProperty("meshOcclusionMaterial");
+
+            // Surface Finding settings
+            useSurfaceFindingSystem = serializedObject.FindProperty("useSurfaceFindingSystem");
+            surfaceFindingPhysicsLayer = serializedObject.FindProperty("surfaceFindingPhysicsLayer");
+            surfaceFindingMinimumArea = serializedObject.FindProperty("surfaceFindingMinimumArea");
+            displayFloorSurfaces = serializedObject.FindProperty("displayFloorSurfaces");
+            floorSurfaceMaterial = serializedObject.FindProperty("floorSurfaceMaterial");
+            displayCeilingSurfaces = serializedObject.FindProperty("displayCeilingSurfaces");
+            ceilingSurfaceMaterial = serializedObject.FindProperty("ceilingSurfaceMaterial");
+            displayWallSurfaces = serializedObject.FindProperty("displayWallSurfaces");
+            wallSurfaceMaterial = serializedObject.FindProperty("wallSurfaceMaterial");
+            displayPlatformSurfaces = serializedObject.FindProperty("displayPlatformSurfaces");
+            platformSurfaceMaterial = serializedObject.FindProperty("platformSurfaceMaterial");
+        }
+
+        public override void OnInspectorGUI()
+        {
+            RenderMixedRealityToolkitLogo();
+            if (!CheckMixedRealityManager())
+            {
+                return;
+            }
+
+            if (GUILayout.Button("Back to Configuration Profile"))
+            {
+                Selection.activeObject = MixedRealityManager.Instance.ActiveProfile;
+            }
+
```

Comment:
Don't forget to add the button to return to the main config profile

Reply:
i took this code from the boundary profile as is... i see the button and have tested that it works.